examples/monitored.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+import logging
+import time
+
+from saq import CronJob, Queue, monitored_job
+
+logger = logging.getLogger("saq")
+logger.setLevel(logging.DEBUG)
+
+
+@monitored_job
+async def monitored_sleeper(ctx):
+    print("executing monitored sleeper job.")
```

Comment:
We create a "logger" above, but then use print.

Maybe we should be using the logger?